Patches/Seventh Seraph/7S_Weapon

```diff
@@ -0,0 +1,373 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+
```

Comment:
This file is missing the .xml file type.

Reply:
Damn. Must've made an error while copy pasting contents of the file and neglected to check for the correct file type. Will edit soon.

Reply:
Mix between industrial and charged ammo type is purely because in Destiny, these guns are divided between Kinetic and Energy weapon types. Kinetic weapons are also described as being older than the energy weapon variants, and so, I believed that Industrial and Charged ammunition in Rimworld would work as an analogue.